lib/rules/order-in-components.js

```diff
@@ -19,11 +19,9 @@ const defaultOrder = [
   'inheritAttrs',
   'model',
   ['props', 'propsData'],
-  'data',
+  ['asyncData', 'fetch', 'data'],
```

Comment:
I suggest moving fetch to the top of `asyncData` because it is more context less

Reply:
@pi0 but as I understand, having asyncData and fetch in same array means the order can be freely chosen between those. Then it shouldn't be need for moving them around?

Reply:
OK, I have actually updated to have them in explicit order.